server/src/main/java/com/hedera/block/server/exception/UnhandledIOException.java

```diff
@@ -1,15 +1,30 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.hedera.block.server.exception;
 
 import java.io.Serial;
 
 /**
- * UnhandledIOException is a RuntimeException that wraps an IOException.
- * This can be used to rethrow checked IOExceptions as unchecked exceptions.
+ * UnhandledIOException is a RuntimeException that wraps an IOException. This can be used to rethrow
```

Comment:
If spotless is forcing this, it's wrong.
Javadoc depends on the newline after the `.` of the first sentence.
Not for this PR, but definitely something to fix very soon so spotless isn't forcing
a misuse of Javadoc.